### ISSUE TITLE
Replaced opportunities

### ISSUE THREAD

**Issue by Rossiar** [AndrewGorton/DigitalMarketplaceAlerter#14]
As a Product Owner
I want to be informed when an opportunity has been replaced, with a link to the replacement
So that I do not pursue opportunities that are no longer relevant.

Depends on #13

**Comment by Rossiar** [comment]
Will likely involve a "replaced" flag, the opportunity template `detailview.ftl`  can detect this and modify content to notify that the opportunity has been replaced and is not current.
